### ISSUE TITLE
Undeclared dependency Safe::Isa

### ISSUE THREAD

**Issue by eserte** [Csson/p5-Dist-Iller#2]
Dist-Iller-0.1201's test suite fails if Safe::Isa is not installed:

```
Can't locate Safe/Isa.pm in @INC (you may need to install the Safe::Isa module) (@INC contains: /home/cpansand/.cpan/build/2015112609/Dist-Iller-0.1201-IY21W5/blib/lib /home/cpansand/.cpan/build/2015112609/Dist-Iller-0.1201-IY21W5/blib/arch /opt/perl-5.23.5/lib/site_perl/5.23.5/x86_64-linux /opt/perl-5.23.5/lib/site_perl/5.23.5 /opt/perl-5.23.5/lib/5.23.5/x86_64-linux /opt/perl-5.23.5/lib/5.23.5 .) at /home/cpansand/.cpan/build/2015112609/Dist-Iller-0.1201-IY21W5/blib/lib/Dist/Iller/Builder.pm line 16.
BEGIN failed--compilation aborted at /home/cpansand/.cpan/build/2015112609/Dist-Iller-0.1201-IY21W5/blib/lib/Dist/Iller/Builder.pm line 16.
Compilation failed in require at t/02-builder.t line 4.
BEGIN failed--compilation aborted at t/02-builder.t line 4.
t/02-builder.t ......... 
Dubious, test returned 2 (wstat 512, 0x200)
No subtests run 
...
```

**Comment by Csson** [comment]
Hi!

Thanks, I will fix.

**Comment by Csson** [comment]
Should be fixed in 0.1202.
